src/theme/prism-include-languages.js

```diff
@@ -10,12 +10,25 @@ import siteConfig from "@generated/docusaurus.config"
 const prismIncludeLanguages = PrismObject => {
   if (ExecutionEnvironment.canUseDOM) {
     const {
-      themeConfig: { prism: { additionalLanguages = [] } = {} },
+      themeConfig: {
+        prism: { additionalLanguages = [], magicComments = [] } = {},
+      },
     } = siteConfig
     window.Prism = PrismObject
     additionalLanguages.forEach(lang => {
       require(`prismjs/components/prism-${lang}`) // eslint-disable-line
     })
+    magicComments.push({
+      className: "docusaurus-highlight-removed-code-lines",
+      line: "Next line is removed",
+      block: { start: "Removed lines start", end: "Removed lines end" },
+    })
+    magicComments.push({
+      className: "docusaurus-highlight-added-code-lines",
+      line: "Next line is added",
+      block: { start: "Added lines start", end: "Added lines end" },
+    })
```

Comment:
```suggestion
    magicComments.push({
      className: "docusaurus-highlight-removed-code-lines",
      line: "removed-line",
      block: { start: "removed-block-start", end: "removed-block-end" },
    })
    magicComments.push({
      className: "docusaurus-highlight-added-code-lines",
      line: "added-line",
      block: { start: "added-block-start", end: "added-block-end" },
    })
```

Are these phrases a convention that we currently follow? I'd suggest commends as `kebab-case` strings to avoid confusion around capitalization and how the command should be used in md files. I'll approve it as this is only a suggestion.

Reply:
fixed :)